circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceTraitCodec.scala

```diff
@@ -136,6 +136,11 @@ trait CirceTraitCodec[Ser <: AnyRef] extends Codec[Ser] {
     }
   }
 
+  /*
+   * Some types from `foundSerializables` might have not been added to `codecs` sequence.
```

Comment:
While you're here... `found...` sounds awkward/unnatural, pls change to sth like `detectedSerializables`